pkgs/development/compilers/llvm/11/compiler-rt/default.nix

```diff
@@ -1,5 +1,8 @@
 { lib, stdenv, llvm_meta, version, fetch, cmake, python3, libllvm, libcxxabi }:
 
+# https://reviews.llvm.org/D43106#1019077
+assert (stdenv.hostPlatform.isRiscV && stdenv.hostPlatform.is32bit) -> stdenv.cc.isClang;
```

Comment:
It's impossible to override derivations with failing asserts #53487. Please use `meta.broken` instead.

Reply:
Wow, TIL. Thanks!

Reply:
Thank you for pointing this out. Lemme know if the changes I just pushed are more like what you had in mind, @veprl.

Reply:
Looks good, thank you